cmd/explaintest/r/explain_easy_stats.result

```diff
@@ -100,20 +100,20 @@ explain select c2 = (select c2 from t2 where t1.c1 = t2.c1 order by c1 limit 1)
 id	parents	children	task	operator info	count
 TableScan_15			cop	table:t1, range:[-inf,+inf], keep order:false	1999.00
 TableReader_16	Apply_14		root	data:TableScan_15	1999.00
-IndexScan_33	Selection_35		cop	table:t2, index:c1, range:[<nil>,+inf], keep order:true	1.25
-Selection_35	Limit_36	IndexScan_33	cop	eq(test.t1.c1, test.t2.c1)	1.00
-Limit_36		Selection_35	cop	offset:0, count:1	1.00
-TableScan_34			cop	table:t2, keep order:false	1.00
-IndexLookUp_37	Limit_21		root	index:Limit_36, table:TableScan_34	1.00
-Limit_21	Apply_14	IndexLookUp_37	root	offset:0, count:1	1.00
+IndexScan_32	Selection_34		cop	table:t2, index:c1, range:[<nil>,+inf], keep order:true	1.25
```

Comment:
Why is the ID changed?

Reply:
due to the refactor of `DataSource.convertToIndexScan()`

Reply:
Got it.